sat/cli/jobstat/main.py

```diff
@@ -0,0 +1,72 @@
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+The main entry point for the jobstat subcommand.
+"""
+import logging
+from sat.apiclient.gateway import APIError, APIGatewayClient
+from sat.report import Report
+from sat.config import get_config_value
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def unload_json(url):
+    """Unloads the JSON object from the server url and formats the data as a dictionary"""
+    try:
+        connect = APIGatewayClient()
```

Comment:
I think this would potentially work though it won't work if the API Gateway is authenticated.

We don't usually instantiate `APIGatewayClient` directly, instead we always create subclasses for the different services.

Here is a gist explaining what I think should change: https://gist.github.hpe.com/eli-kamin/3675a843e0b343f3360f2bcd11327da1